README.md

```diff
@@ -200,7 +202,8 @@ They should be set via `Prettier`'s `overrides` option
   - `'trim-all'` -> Trim all spaces within comments.  
     Example: `// this is a comment`
 
-- `pugSortAttributesBeginning` (and `pugSortAttributesEnd`)
+- `sortAttributesBeginning` | `pugSortAttributesBeginning` 
```

Comment:
There is (intentionally) no `sortAttributesBeginning` or `sortAttributesEnd`
I also think about to deprecate all previous options without `pug` prefix that are not a prettier default option

Reply:
Ah right, you mentioned that! Will remove it. Shall I remove the other also, so people stop using them?

Reply:
I think yes, we should remove the non alias ones (pug-specific-option)

Reply:
Ok, change coming up!

Reply:
But then the whole part about __They should be set via `Prettier`'s `overrides` option__ can be left out, right? Making docs simpler, a good thing!

Reply:
Yes, this also can be upgraded to just use the new pug prefixes

You only need to mention specifically that `singleQuote` will be the opposite of what prettier attributes will be quoted
Cause the js code in values of pug attributes uses these quotes

Reply:
Yes makes sense!